x-pack/solutions/security/packages/kbn-scout-security/src/playwright/fixtures/worker/apis/detection_rule.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { KbnClient, measurePerformanceAsync, ScoutLogger } from '@kbn/scout';
+import { ScoutSpaceParallelFixture } from '@kbn/scout/src/playwright/fixtures/worker';
```

Comment:
can be imported directly from `'@kbn/scout`

Reply:
Is not allowing me to import it directly from `@kbn/scout`, I'm seeing the following error: `'"@kbn/scout"' has no exported member named 'ScoutSpaceParallelFixture'. Did you mean 'ScoutParallelTestFixtures'?`

Reply:
Then let's use `ScoutParallelTestFixtures[scoutSpace]`. We only pass the value for parallel run, it might even better from explanation perspective.